monkestation/code/modules/surgery/organs/internal/tongue.dm

```diff
@@ -92,3 +92,207 @@
 		RND_CATEGORY_CYBERNETICS + RND_SUBCATEGORY_CYBERNETICS_ORGANS_1
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_MEDICAL | DEPARTMENT_BITFLAG_SCIENCE
+
+/obj/item/organ/internal/tongue/robot/polyglot_voicebox
+	name = "polyglot voicebox"
+	desc = "A voice synthesizer that allows you to emulate the tongues of other species."
+	say_mod = "beeps"
+	//The current tongue being emulated.
+	var/current_tongue = "Synth"
+	var/datum/action/innate/select_tongue/select_tongue
+	var/draw_length = 3
+	var/chattering = FALSE
+	var/phomeme_type = "sans"
+	var/list/phomeme_types = list("sans", "papyrus")
```

Comment:
if this list is never actually changed at any point, this can also be a static var